_posts/2023-09-26-Hacktoberfest-2023.adoc

```diff
@@ -0,0 +1,53 @@
+
+== Get started with our projects
+
+This list contains some really interesting projects that are under the WildFly umbrella, waiting for your magic touch. There’s something for everyone from Java and Ansible to TypeScript and JavaScript.
+
+. *WildFly:* WildFly is a powerful, modular, and lightweight application server that empowers you to build amazing applications.
+* Here is the https://github.com/wildfly/wildfly[GitHub repository], https://github.com/wildfly/wildfly/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/browse/WFLY-17539?filter=12421535[Hacktoberfest Issues link].
+
+. *WildFly Elytron:* WildFly Elytron is a new WildFly sub-project that is completely replacing the combination of PicketBox and JAAS as the WildFly client and server security mechanism.
+* Here is the https://github.com/wildfly-security/wildfly-elytron[GitHub repository], https://github.com/wildfly-security/wildfly-elytron/blob/2.x/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/browse/ELY-2636?filter=12383825[Hacktoberfest Issues link].
+
+. *Ansible Middleware:* Ansible Middleware is a suite of supported content that combines the power of Ansible automation with Red Hat's Middleware portfolio. It allows organizations to meet their business demands and automate the installation, configuration, and management of Enterprise Middleware.
+* Here is the https://github.com/ansible-middleware[GitHub repositories], https://github.com/ansible-middleware/keycloak/blob/main/CONTRIBUTING.md[Contributing Guide], and https://github.com/search?q=org%3Aansible-middleware+state%3Aopen+label%3Ahacktoberfest&type=issues[Hacktoberfest Issues link].
+
+. *Undertow:* Undertow is a versatile and high-performance web server that provides both blocking and non-blocking APIs based on NIO. Its modular architecture allows you to build a web server tailored to your needs, from a full Java EE servlet 4.0 container to a low-level non-blocking handler.
+* Here is the https://github.com/undertow-io/undertow[GitHub repository], https://github.com/undertow-io/undertow/blob/master/CONTRIBUTING.md[Contributing Guide], and Hacktoberfest Issues link.
+
+. *Narayana:* Narayana is a transactions toolkit that provides support for applications using various standards-based transaction protocols. It's an essential component of the WildFly application server and continues to be developed as a standalone transaction manager.
+* Here is the https://github.com/jbosstm/narayana[GitHub repository], https://github.com/jbosstm/narayana/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/browse/JBTM-2493?filter=12421681[Hacktoberfest Issues link].
+
+. *Infinispan Console:* Infinispan Console is a front-end application for the web console of the new Infinispan Server. Built using Patternfly 5 and React 18, it's a dynamic and engaging project to contribute to.
```

Comment:
s/new Infinispan/Infinispan

Per https://github.com/infinispan/infinispan-console/releases/tag/0.1.Final this dates back to late 2019.